lapis2/src/test/kotlin/org/genspectrum/lapis/request/AminoAcidMutationTest.kt

```diff
@@ -80,6 +80,10 @@ class AminoAcidMutationTest {
                     "\"gENe1:123A\"",
                     AminoAcidMutation("gene1", 123, "A"),
                 ),
+                Arguments.of(
+                    "\"MAYBE(gene1:123A)\"",
```

Comment:
Only out of curiosity: when I type `MAYBE(gene1:G123A)` this would also be a valid query? If yes, I would add it here to the test. (I mean we ignore the symbol before 123, but it would be nice to have it explicit here)